packages/renderer/src/lib/ui/DropdownMenu.svelte

```diff
@@ -41,7 +41,8 @@ function onWindowClick(e: any) {
       toggleMenu();
     }}"
     bind:this="{outsideWindow}"
-    class="mr-2 text-gray-400 hover:bg-charcoal-800 hover:text-purple-400 font-medium rounded-md inline-flex items-center px-2 py-2 text-center">
+    class="mr-2 text-gray-400 hover:bg-charcoal-800 hover:text-purple-400 font-medium rounded-md inline-flex items-center px-2 py-2 text-center"
+    aria-label="Dropdown menu">
```

Comment:
How is this change related to registries?

Reply:
For each registry row there's a dropdown menu to offer the edit and delete options.

Reply:
@xbabalov ok, thanks!